backends-velox/src/test/scala/io/glutenproject/execution/TestOperator.scala

```diff
@@ -90,6 +90,12 @@ class TestOperator extends WholeStageTransformerSuite {
     val df = runQueryAndCompare("select * from temp_test_is_null where col1 is null") { _ => }
     checkLengthAndPlan(df, 2)
   }
+  test("velox parquet write") {
+    val df = runQueryAndCompare(
+      "select * from lineitem where l_comment is not null " +
+        "and l_orderkey = 1") { _ => }
+    df.write.mode("append").format("velox").save("file:///tmp/velox_parquet_write/")
```

Comment:
Does this file will exists in disk after test finished? How can like Spark suite write to SparkWarehouse

Reply:
Updated.